.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "useTabs": false,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "jsxBracketSameLine": false,
```

Comment:
@ayusharma it's ok?

Reply:
Let's keep trailing commas. Generates small changes in PRs.

Reply:
What's about `jsxBracketSameLine` ?

Reply:
Let's keep it `false`.